src/main/java/core/basesyntax/EntryToFile.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class EntryToFile {
```

Comment:
```suggestion
public class WriteToFile {
```

or similar, because EntryToFile a little bit confusing